src/Appwrite/Event/Audit.php

```diff
@@ -108,6 +108,29 @@ public function getIP(): string
         return $this->ip;
     }
 
+    /**
+     * Set description for this audit event
+     *
+     * @param string $event
+     * @return self
+     */
+    public function setEvent(string $event): self
+    {
+        $this->event = $event;
+
+        return $this;
+    }
+
+    /**
+     * Returns the  audit event description.
+     *
+     * @return string
+     */
+    public function getEvent(): string
+    {
+        return $this->event;
+    }
+
```

Comment:
This is already part of the parent class 👍🏻

Reply:
So we don't need it here.

Reply:
https://github.com/appwrite/appwrite/blob/342c4a3253816b9fb72b6ede26a617e31e0c362e/src/Appwrite/Event/Event.php#L81-L101

Reply:
Cool